variable/technology/technologies.yaml

```diff
@@ -460,3 +531,326 @@ Investment|Energy Supply|Liquids|Oil:
                 capturing equipment should be included but not the one on
                 transport and storage.
    unit: billion US$2010/yr
+
+Line Type|Electricity|Transmission|Line:
+   description: Line type: alternating current (AC) and direct current (DC)
+   unit: [AC,DC]
+
+Line Losses|Electricity|Transmission|Line:
+   description: Half ohmic losses of a line
+   unit: GW
+
+Linear Variable Cost|Electricity|Coal:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Linear Variable Cost|Electricity|Gas:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Linear Variable Cost|Electricity|Nuclear:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Linear Variable Cost|Electricity|Oil:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Loss Factor|Electricity|Transmission|Line:
+   description: Transmission losses equal to the line flow times this factor
+   unit: p.u.
+
+Maximum Charge|Electricity|Energy Storage System:
+   description: Maximum charge when storing energy the energy storage system unit.
+   unit: MW
+
+Maximum Storage|Electricity|Energy Storage System:
+   description: Maximum energy that can be stored by the energy storage system unit.
+   unit: TWh
+
+Minimum Storage|Electricity|Energy Storage System:
+   description: Minimum energy that can be stored by the energy storage system unit.
+   unit: TWh
+
+Minimum Down Time|Electricity:
```

Comment:
This contradicts with the variables `Minimum Off Duration` introduced by @sandrinecharousset in #24

Reply:
openTEPES follows the variables names of the unit commitment problem from:
> D. Rajan and S. Takriti, “Minimum up/down polytopes of the unit commitment problem with start-up costs,” IBM, New York, Technical Report RC23628, 2005. https://pdfs.semanticscholar.org/b886/42e36b414d5929fed48593d0ac46ae3e2070.pdf

But, `Minimum Off Duration` and `Minimum On Duration` could be more intuitive.